js2.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const btn = document.getElementById('btn');
+
+//整数チェック
+const checkNum = function (x, y) {
+    if ((Number.isSafeInteger(x) === true) && (Number.isSafeInteger(x) === true)) {
+        return true;
+    } else {
+        return false;
+    }
+}
+
+const ul = document.createElement('ul');
+
+//計算機能
+const outResult = function () {
+    const fizzValue = document.getElementById('fizz').value;
+    const buzzValue = document.getElementById('buzz').value;
+
+    //文字列数値化
+    const fizz = Number(fizzValue);
+    const buzz = Number(buzzValue);
+
+    if ((fizz === 0) || (buzz === 0)) {　//空欄なら
+        const li = document.createElement('li');
+        li.textContent = '整数値を入力してください';
+        ul.appendChild(li);
+    } else if ((fizz >= 100) && (buzz >= 100)) {
```

Comment:
`return` を用いて、なるべく `else` が続かないようにした方がコードが見やすくなりますよ。

```suggestion
        ul.appendChild(li);
        return;
    }
    if ((fizz >= 100) && (buzz >= 100)) {
```